src/wiki/development/build-instructions.md

```diff
@@ -166,36 +166,56 @@ We recommend using a build workflow based on MSYS2, as it's the easiest way to g
 ### Dependencies
 
 - [MSYS2](https://www.msys2.org/) - Software Distribution and Building Platform for Windows
-  - Make sure to follow all instructions on the webpage.
 - [Java JDK 8 or later](https://adoptium.net/)
   - Make sure that "Set JAVA_HOME variable" is enabled in the Adoptium installer.
 
 ### Preparing MSYS2
 
-1. Open either the *MSYS2 MinGW CLANG x64* (reccomended), the *MSYS2 MinGW UCRT x64*, the *MSYS2 MinGW x64*, the *MSYS2 MinGW CLANG x86* or the *MSYS2 MinGW x86* shortcut from the start menu
+1. Open one of the shortcuts from the MSYS2 folder in the Start menu
 
-   - We recommend building using the CLANG msystems of MSYS2, as they compile considerably faster and with a few less bugs.
+   - We recommend building using the CLANG64 msystems of MSYS2, as they compile considerably faster and with a few less bugs.
```

Comment:
*the* clang64 *msystems*?

Reply:
That was what the original instructions said. Tbh, I don't even know if CLANG64 is any better than the other systems. I used MinGW64 previously with no issues.